### ISSUE TITLE
Revert adreno_tz: Fix GPU target frequency calculation for high refresh rates 

### ISSUE THREAD

**Issue by radmerti** [AOSPA/android_kernel_oneplus_sm8150#2]
tldr: revert commit b315ca1 because it's wrong and useless

The change should not have an effect on smoothness since the GPU will boost to 100% when it's busy according to that function. And in the other case, e.g. where this commit changes the code, there is no reason to believe that the so called  "black box" algorithm should consider the refresh rate. The algorithm gets the total time and busy time and hence has the relative fraction the GPU has been busy since the last frequency decision. This is completely independent of the display refresh rate.

If anything the higher display refresh rate will lead to more GPU load, which will lead to a higher busy time, which will lead to a higher frequency selected. Exactly what we want.

**Comment by timocapa** [comment]
Ack. Forgot to reply here, will be gone in Uvite due to a new kernel. This kernel isn't touched much anymore

**Comment by radmerti** [comment]
Good to know. Thanks for the reply!

**Comment by radmerti** [comment]
Closing the issue.
